imap-codec/src/lib.rs

```diff
@@ -136,5 +137,6 @@ pub mod fuzz {
 }
 
 pub use codec::*;
+pub use fragmentizer::*;
```

Comment:
Are you sure about this?

- `fragmentizer` contains types with really generic names
- `fragmentizer` is only relevant for decoding

Alternatives that I like:

- Public module `imap_codec::fragmentizer`
- Public module `imap_codec::decode::fragmentizer`
- One of the options above and re-export only `imap_codec::Fragmentizer`

Reply:
Let me think about it more. Didn't gave it enough thought.